rockhound/prem.py

```diff
@@ -0,0 +1,51 @@
+"""
+Load the Preliminary Reference Earth Model (PREM) dataset.
+"""
+import numpy as np
+import pandas as pd
+from .registry import REGISTRY
+
+
+def fetch_prem(load=True):
+    """
```

Comment:
```suggestion
    r"""
```

You need to make this a raw string in order to have `\` in there. I think that's what pylint is complaining about.